src/SimpleGraphs/SimpleGraphs.jl

```diff
@@ -33,8 +34,9 @@ export AbstractSimpleGraph, AbstractSimpleEdge,
     #smallgraphs
     smallgraph,
     # Euclidean graphs
-    euclidean_graph
-
+    euclidean_graph,
+    #overrides
+    common_neighbors, intersect
```

Comment:
It is not necessary to export overwritten symbols.
We could still do it though, so it works as a documentation.

Reply:
Yes, I did that so that it is easy to remember that the method is overridden, can remove it though.